src/service/mobile_config.proto

```diff
@@ -104,17 +111,19 @@ message router_list_res_v1 {
 }
 
 message admin_add_key_req_v1 {
```

Comment:
when adding a key the `key_role` is required to be supplied, can one key be added to more than one role ?

I note when removing a key, no `key_role` is required in the request....should it be ?